### PR TITLE
Added Hungarian (hu) translation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by hbjydev** [comment]
@huszakgyozo Please do some testing of your i18n file with this new branch target.

**Comment by ghost** [comment]
> @huszakgyozo Please do some testing of your i18n file with this new branch target.

All right. Do I need to modify the config.toml and vercel.json files?

**Comment by FoggyMtnDrifter** [comment]
@huszakgyozo Please follow [this wiki article](https://wiki.rockylinux.org/en/Web/Website/contrib-i18n) and update the required files accordingly to test and then commit your changes to those files to the new target branch. In the meantime, I'll mark this PR as a draft and you can mark it ready for review when you've made the changes.

File: i18n/hu.toml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  [landing.p2.content]
+  other = "Rocky Linux a közösség által működtetett vállalati környezetbe szánt operációs rendszer, amelyet úgy terveztek, hogy 100%-os hibakeresés és kompatibilitás érdekében kompatibilis legyen az Amerikai Enterprise Linux rendszerrel most, amikor a CentOS irányt váltott. A közösség intenzív fejlesztés alatt áll. A Rocky Linuxot Gregory Kurtzer, a CentOS projekt alapítója vezeti. A kiadáshoz nincs ETA. Kérjük a közreműködőket, hogy vegyék fel a kapcsolatot velünk az ezen az oldalon kínált kommunikációs lehetőségekkel."
+
+[faq]
+  [faq.title]
+  other = "Gyakran Ismételt Kérdések"
+
+  [faq.q]
+    [faq.q.downstream-partner-direction]
+    other = "Mit értesz azalatt, \" hogy ez a downstream partner irányt váltott?\""
```
Comment (SergeCroise) [inline i18n/hu.toml:73]: "Mit értesz azalatt, \\"hogy ez a downstream partner irányt váltott\\"?"

**Review by SergeCroise — CHANGES_REQUESTED** [review]

**Comment by hbjydev** [comment]
I'm now at a point where I don't want to accept any more i18n PRs, simply because it's going to become FAR too unmaintainable, so I'm going to prioritise #134 and close this PR.

**Pull request closed without merging**